### PR TITLE
More CLI Bootstrap fixes for macOS

### PR DESCRIPTION
**Pull request by OR13**
When updating documentation I encountered a number of problems.

### CODE REVIEW

File: packages/transmute-cli/src/commands/debug/debug [hunk 1]
```diff
@@ -0,0 +1,5 @@
+echo 'DEBUG: CLI Details'
+echo $(whoami)
+echo $PWD
+
```
Comment (eolszewski) [inline packages/transmute-cli/src/commands/debug/debug]: Can we make a note here about the code-splitting that there will be other debug commands in the future?

File: packages/transmute-cli/src/commands/provision/index.js [hunk 2]
```diff
@@ -17,7 +17,13 @@ module.exports.minikube = (dryrun, clusterName, minikubeDriver) => {
     ' --memory ' +
     MINIKUBE_MEMORY;
   +' --profile ' + MINIKUBE_PROFILE;
-  let prov_cmd = minikube_start + ' --vm-driver=virtualbox';
+
+  if (minikubeDriver == 'none') {
+    minikube_start = 'sudo ' + minikube_start;
```
Comment (eolszewski) [inline packages/transmute-cli/src/commands/provision/index.js]: Should we provide a message before this executes to let the user know why they are going to have to input their password?

Comment (OR13) [reply packages/transmute-cli/src/commands/provision/index.js]: yes

File: packages/transmute-cli/src/commands/debug/debug (hunk 1 above)
Comment (OR13) [reply packages/transmute-cli/src/commands/debug/debug]: yes

**Pull request merged**